server/lib/cleanup/languages/fr.js

```diff
@@ -49,9 +51,134 @@ function clean(sentences) {
+	  .replace(/(^|\s)XV(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' quinzième siècle ')
+	  .replace(/(^|\s)XVI(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' seixième siècle ')
+	  .replace(/(^|\s)XVII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dix-septième siècle ')
+	  .replace(/(^|\s)XVIII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dix-huitième siècle ')
+	  .replace(/(^|\s)(XIX|XVIIII)(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' dix_neuvième siècle ')
+	  .replace(/(^|\s)XX(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' vingtième siècle ')
+	  .replace(/(^|\s)XXI(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' vingt-et-unième siècle ')
+	  .replace(/(^|\s)XXII(e|è)(me)? s.(\s|\.|,|\?|!|$)/g, ' vingt-deuxième siècle ')
+      
+     //roman numerals.
+	  .replace(/(^|\s)I(\s|\.|,|\?|!|$)/g, ' premier ') //translated as 'first'. We considere that it's encountered after chapter (Chapitre I => 'chapitre premier'). Work also with names (Charles I => 'Charles premier')
+	  .replace(/(^|\s)II(\s|\.|,|\?|!|$)/g, ' deux ')
+	  .replace(/(^|\s)III(\s|\.|,|\?|!|$)/g, ' trois ')
+	  .replace(/(^|\s)IV(\s|\.|,|\?|!|$)/g, ' quatre ')
+	  .replace(/(^|\s)V(\s|\.|,|\?|!|$)/g, ' cinq ')
+	  .replace(/(^|\s)VI(\s|\.|,|\?|!|$)/g, ' six ')
+	  .replace(/(^|\s)VII(\s|\.|,|\?|!|$)/g, ' sept ')
+	  .replace(/(^|\s)VIII(\s|\.|,|\?|!|$)/g, ' huit ')
+	  .replace(/(^|\s)(VIIII|IX)(\s|\.|,|\?|!|$)/g, ' neuf ')
+	  .replace(/(^|\s)X(\s|\.|,|\?|!|$)/g, ' dix ')
+	  .replace(/(^|\s)XI(\s|\.|,|\?|!|$)/g, ' onze ')
+	  .replace(/(^|\s)XII(\s|\.|,|\?|!|$)/g, ' douze ')
+	  .replace(/(^|\s)XIII(\s|\.|,|\?|!|$)/g, ' treize ')
+	  .replace(/(^|\s)(XIIII|XIV)(\s|\.|,|\?|!|$)/g, ' quatorze ')
+	  .replace(/(^|\s)XV(\s|\.|,|\?|!|$)/g, ' quinze ')
+	  .replace(/(^|\s)XVI(\s|\.|,|\?|!|$)/g, ' seize ')
+	  .replace(/(^|\s)XVII(\s|\.|,|\?|!|$)/g, ' dix-sept ')
+	  .replace(/(^|\s)XVIII(\s|\.|,|\?|!|$)/g, ' dix-huit ')
+	  .replace(/(^|\s)(XIX|XVIIII)(\s|\.|,|\?|!|$)/g, ' dix-neuf ')
+	  .replace(/(^|\s)XX(\s|\.|,|\?|!|$)/g, ' vingt ')
+ 	  .replace(/(^|\s)XXI(\s|\.|,|\?|!|$)/g, ' vingt-et-un ')
+ 	  .replace(/(^|\s)XXII(\s|\.|,|\?|!|$)/g, ' vingt-deux ')
+      
+      //first, second, etc.
+	  .replace(/(^|\s)1er?s?(\s|\.|,|\?|!|$)/g, ' premier ')
+	  .replace(/(^|\s)1(e|è)res?(\s|\.|,|\?|!|$)/g, ' premier ')
+	  .replace(/(^|\s)2(e|è)?me?s?(\s|\.|,|\?|!|$)/g, ' deuxième ')
+	  .replace(/(^|\s)2n?ds?(\s|\.|,|\?|!|$)/g, ' second ')
+	  .replace(/(^|\s)2n?des?(\s|\.|,|\?|!|$)/g, ' seconde ')
+	  .replace(/(^|\s)3i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' troisième ')
+	  .replace(/(^|\s)4i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' quatrième ')
+	  .replace(/(^|\s)5i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' cinquième ')
+	  .replace(/(^|\s)6i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' sixième ')
+	  .replace(/(^|\s)7i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' septième ')
+	  .replace(/(^|\s)8i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' huitième ')
+	  .replace(/(^|\s)9i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' neuvième ')
+	  .replace(/(^|\s)10i?(e|è)me?s?(\s|\.|,|\?|!|$)/g, ' dixième ')
```

Comment:
These can be removed, as numbers are not allowed anyway in the Sentence Collector.